dowhy/causal_refuters/add_unobserved_common_cause.py

```diff
@@ -43,6 +43,9 @@ def refute_estimate(self):
             new_effect = new_estimator.estimate_effect()
             refute = CausalRefutation(self._estimate.value, new_effect.value,
                                     refutation_type="Refute: Add an Unobserved Common Cause")
+            
+            refute.new_effect = np.array(new_effect.value)
+            
```

Comment:
since this is the main method of the class and has got a bit complicated, it will be nice if you can add a docstring for this function in the sphinx format (you can look at the docstring for any other method as a reference)

Reply:
I added the docstring for the unobserved common cause for both the refute_estimate and the include_confounders_effect functions.